src/test/regress/sql/disable_autovacuum.sql

```diff
@@ -1,2 +1,8 @@
 alter system set autovacuum = off;
 select gp_segment_id, pg_reload_conf() from gp_id union select gp_segment_id, pg_reload_conf() from gp_dist_random('gp_id');
+-- start_ignore 
+-- The reason for restarting cbdb here is that if the subsequent 
+-- vacuum ao test(uao*_catalog_tables/threshold) encounters an
+-- unfinished transaction, it will fail.
+\!gpstop -ari
+-- end_ignore
```

Comment:
For a parallel mode test(make installcheck-cbdb-parallel), there may be multiple processes running.
Does this restart cluster have no impact on them?

Reply:
in `src/test/regress/parallel_schedule`

```

# ----------
# Because vacuum will detect concurrently running transactions, it is necessary to
# run this test on its own, or it might not detect the pages are 'all visible'
# ----------
test: disable_autovacuum
test: vacuum_stats
test: enable_autovacuum
```

`disable_autovacuum` will run alone, I think it will have not impact for other case.

also tried this change in CI(icw-parallel), no found any failed in this case.

Reply:
Seems no impact because of serial run. If no further comments, I will resolve conversation.